Chart: Add Getting Started topic (#7613)
DevExpress/devextreme-documentation: concepts/05 UI Components/Chart/00 Getting Started with Chart/25 Add and Customize Tooltips.md

> To add tooltips to Chart, assign `true` to the **tooltip**.[enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#enabled) property. Default tooltips display values when users hover over points. To implement custom tooltips, define the [customizeTooltip](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#customizeTooltip) function. This example uses **customizeTooltip** to display the difference between the current and average `actualSpend` values (calculated in the `calculateAverageSpend()` method):

Reviewer: ```suggestion
To add tooltips to a Chart, assign `true` to the **tooltip**.[enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#enabled) property. Default tooltips display values when users hover over points. To implement custom tooltips, define the [customizeTooltip](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#customizeTooltip) function. This example uses **customizeTooltip** to display the difference between current and average `actualSpend` values (calculated in the `calculateAverageSpend()` method):
```

> To add tooltips to Chart, assign `true` to the **tooltip**.[enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#enabled) property. Default tooltips display values when users hover over points. To implement custom tooltips, define the [customizeTooltip](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#customizeTooltip) function. This example uses **customizeTooltip** to display the difference between `actualSpend` points and the average value of all `actualSpend` fields (calculated in the `calculateAverageSpend()` method):

Reviewer 2: ```suggestion
To add tooltips to Chart, assign `true` to the **tooltip**.[enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#enabled) property. Default tooltips display values when users hover over points. To implement custom tooltips, define the [customizeTooltip](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#customizeTooltip) function. This example uses **customizeTooltip** to display the difference between the current and average `actualSpend` values (calculated in the `calculateAverageSpend()` method):
```
